packages/contentful--react-apps-toolkit/src/useFieldValue.tsx

```diff
@@ -34,7 +34,7 @@ export function useFieldValue<Value = unknown>(
   const updateValue = useCallback(
     async (newValue: Value | undefined) => {
       setValue(newValue);
-      await entryFieldApi.setValue(newValue, localeWithDefault);
```

Comment:
can we not just return without the await and remove the `async` from the function?

Reply:
See this example. Removing the `await` from `A()` will remove `A` from the stack tracek

```js
async function A() {
  return await B();
}

async function B() {
  await Promise.resolve();
  throw new Error('BEEP BEEP');
}

A();

```